backends/gdx-backends-gwt/src/com/badlogic/gdx/backends/gwt/preloader/AssetDownloader.java

```diff
@@ -143,7 +144,14 @@ public void onSuccess (Blob result) {
 	}
 
 	public void loadImage (final String url, final String mimeType, final AssetLoaderListener<ImageElement> listener) {
-		loadImage(url, mimeType, null, listener);
+		String crossOrigin = null;
+
+		// Enable CORS if we're running from a different URL to the host page
+		if (!url.startsWith(GWT.getHostPageBaseURL())) {
+			crossOrigin = "anonymous";
+		}
+
+		loadImage(url, mimeType, crossOrigin, listener);
```

Comment:
This is good, however, I wonder why only images are concerned and why text files and other assets are not.

Reply:
The main issue is that WebGL cannot access image assets without cross-origin, other asset types (audio, text files, shaders, etc) correctly resolve.